site/content/docs/integrations/terraform.md

```diff
@@ -197,6 +197,10 @@ We often publish in-depth articles on how to use Terraform with Checkly on our b
 1. [Managing Checkly checks with Terraform](https://blog.checklyhq.com/managing-checkly-checks-with-terraform/)
 2. [Scaling Puppeteer & Playwright on Checkly with Terraform](https://blog.checklyhq.com/scaling-puppeteer-playwright-on-checkly-with-terraform/)
 
+
+>**Error: 402 payment required** If you find this error when creating checks through Terraform or the API, possibly you have reached the maximum number of checks you can create on your account. Contact Checkly Support to talk about options. 
```

Comment:
Given this will happen in the "terraform apply" step, I feel we should place the notice in the section above, i.e. at the end of the "Applying changes" section of this page.